internal/slicer/slicer.go

```diff
@@ -36,21 +36,15 @@ func Run(options *RunOptions) error {
 		if path[0] != '/' {
 			panic("bug: tried to add relative path to known paths")
 		}
-		cleanPath := filepath.Clean(path)
-		slashPath := cleanPath
-		if path[len(path)-1] == '/' && cleanPath != "/" {
-			slashPath += "/"
-		}
+		path = fsutil.Clean(path)
```

Comment:
The output of these two snippets are not identical I believe?

```slashpath``` will always end with a ```Separator```.
```path``` uses ```fsutil.Clean()``` only, which could end without a ```Separator```.

Reply:
Can you elaborate on why you think `slashpath` will always end with separator? From diff snippet it looks like it'll only end with separator when path ends with `/`'.

Reply:
Thanks for the find. I didn't notice the bug in `Clean()` (now `SlashedPathClean()`). I've fixed the bug there and added test cases for root directory. Please resolve this thread if appropriate.